Kitodo/src/test/java/de/sub/goobi/helper/encryption/MD4Test.java

```diff
@@ -38,16 +37,12 @@
     }
 
     @Test
-    public void encryptTest() {
+    public void encryptTest() throws Exception {
```

Comment:
Why did you not use more specific `UnsupportedEncodingException` instead of the really general `Exception`?